packages/mui-lab/src/Masonry/Masonry.js

```diff
@@ -1,17 +1,16 @@
-import * as React from 'react';
-import PropTypes from 'prop-types';
-import clsx from 'clsx';
+import { unstable_composeClasses as composeClasses } from '@mui/core';
+import { styled, useThemeProps } from '@mui/material/styles';
 import {
   createUnarySpacing,
   getValue,
   handleBreakpoints,
```

Comment:
(not related to this PR) This is the first time I encounter these functions and I can't figure out by their names what they are supposed to do. @mnajdova how would you feel about giving them more meaningful names?

Reply:
Open to idea, maybe `applyBreakpoints` is better name :) Let's just make sure it is not a public API before renaming